shared/src/vyxal/Elements.scala

```diff
@@ -2492,19 +2492,23 @@ object Elements:
       case a: VNum => a ** 3
       case a: String => VList.from(a.grouped(3).toSeq)
     },
-    addPart(
+    addPartialVect(
       Monad,
       "ɾ",
       "Inclusive One Range | Uppercase",
       List("one->n", "one-range", "to-upper", "upper", "uppercase"),
-      true,
       "a: num -> [1..a]",
+      "a: lst[num] -> apl-style iota from 1 to a",
       "a: str -> a.upper()",
     ) {
       case a: VNum => NumberHelpers.range(1, a)
-      case a: String => a.toUpperCase
+      case a: VList if a.forall(_.isInstanceOf[VNum]) =>
+        NumberHelpers.range(1, a.map(_.asInstanceOf[VNum]))
+      case a => (Monad.vectorise("ɾ") {
```

Comment:
This makes a new function every time. I'd recommend putting something like this in `StringHelpers`:

```scala
val uppercase = Monad.vectorise("ɾ") {
  case a: String => a.toUppercase
}
```

Reply:
In fact, you don't even need to manually vectorise. Since you have that guard up there to only catch `VList`s that only have numbers, you're all good. Just make this last case `case a: String => a.toUpperCase` rather than `case a => ...` (and go back to `addPart`, I think)